src/coreclr/.nuget/optdata/optdata.proj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.Build.NoTargets">
 
   <PropertyGroup>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
```

Comment:
You can remove the line below (DisableIMplicitFrameworkReferences) as that's the default value when using the  Microsoft.Build.NoTargets SDK.